htdocs/product/class/product.class.php

```diff
@@ -1695,7 +1695,7 @@ function updatePrice($newprice, $newpricebase, $user, $newvat='',$newminprice=0,
 				$price = price2num($newprice) / (1 + ($newvat / 100));
 				$price = price2num($price,'MU');
 
-				if ($newminprice != '' || $newminprice == 0)
+				if (($newminprice !== '' && is_numeric($newminprice)) || $newminprice == 0)
```

Comment:
$newminprice may be a non numeric. The goal of function price2num that come after is to transform a non numeric for php value into a numeric php
For example   '1.123,4'  (this is a numeric for human, not for php)
So old code is better. What return a var_dump on $newminprice when you have this warning ?